src/Illuminate/Http/Request.php

```diff
@@ -630,7 +630,7 @@ public function accepts($contentTypes)
             }
 
             foreach ($types as $type) {
-                if ($this->matchesType($accept, $type) || $accept === strtok('/', $type).'/*') {
+                if ($this->matchesType($accept, $type) || $accept === strtok($type, '/').'/*') {
```

Comment:
All the doc i see only suggested `strtok($string, $separator)`. It seem current test are working/green due to preg_match('/application\/.+\+*/', 'application/json')` return true.